### ISSUE TITLE
Focus order is wrong on Order History page

### ISSUE THREAD

**Issue by kacperknapik** [SAP/spartacus#7459]
### Problem
Focus (tabbing) order should go from left to right, top to bottom. Currently the order goes like this: 1, 3, 2 (see the screenshot below)

### Solution
Tabbing should go naturally and be: 1, 2, 3 (see the screenshot below). The HTML implementation of tabs should be changed - tabbing order reflects DOM order, so the buttons (the tabs) should be first, the contents of the tabs should go next.

![image](https://user-images.githubusercontent.com/22354613/81079369-c44e1180-8eef-11ea-8d04-1818c24f44eb.png)
